src/decaylanguage/data/decfile.lark

```diff
@@ -55,17 +55,13 @@ model_label : LABEL
 
 model_alias : "ModelAlias" model_label model
 
-model : (model_label _SEMICOLON | MODEL_NAME_AND_SC | MODEL_NAME_AND_WS model_options _SEMICOLON+ | MODEL_NAME_MULTILINE model_options _SEMICOLON+)
+model : (model_label  | MODEL_NAME model_options?) _SEMICOLON+
```

Comment:
Excellent job! I'm impressed with the simplification achieved here, especially that I remember playing quite some time with the grammar to be able to support everything as before and parse the multi-line model PTO3P!

Reply:
Out of curiosity, you simply used timeit to check the improvement in parsing speed? 15% is already very nice.

Reply:
To be checked but I think you can now also remove the line `%import common.NEWLINE` below.

Reply:
> Excellent job! I'm impressed with the simplification achieved here, especially that I remember playing quite some time with the grammar to be able to support everything as before and parse the multi-line model PTO3P!

Thank you! I think the issue was mostly caused by me not understanding Python regex syntax well enough when changing the Lark grammar last year so the simplification was only really possible by partially reverting this change.

> Out of curiosity, you simply used timeit to check the improvement in parsing speed? 15% is already very nice.

Exactly, I just ran timeit with 50 repetitions on the LHCb / Belle II main decfiles. The results were pretty similar for both files (not surprising I guess since the relevant `Transformer` just visited each model terminal).

> To be checked but I think you can now also remove the line `%import common.NEWLINE` below.

You're right, I've removed it and all tests still ran through without problems.